Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "bech32"
 version = "0.9.1"
-authors = ["Clark Moody", "The rust-bitcoin developers"]
+authors = ["Clark Moody", "Andrew Poelstra", "Tobin Harding", "The rust-bitcoin developers"]
 repository = "https://github.com/rust-bitcoin/rust-bech32"
 documentation = "https://docs.rs/bech32/"
-description = "Encodes and decodes the Bech32 format"
+description = "Encodes and decodes the Bech32 format (bech32 and bech32m)"
 readme = "README.md"
-keywords = ["base32", "encoding", "bech32"]
+keywords = ["base32", "encoding", "bech32", "bech32m"]
```

Comment:
Maybe also add "bitcoin" and "cryptocurrency" here. I forget if there's a limit on the number of keywords you can have.

Reply:
Added. I added this too:
`categories = ["encoding", "cryptography::cryptocurrencies"]`